### PR TITLE
Bryan's README corrections

### PR DESCRIPTION
**Pull request by ineiti**
Some README-changes proposed by Bryan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,23 @@ The `cosi` application provides both a client for signing messages,
...
+If you compiled from source, please change the directory like so:
+
+```bash
+cd $GOPATH/src/github.com/dedis/cosi
+```
+
+If you used the binary distribution, please use
+
+```bash
+cd $( dirname $( which cosi ) )
```
Comment (nikkolasg) [inline README.md:95]: If ~/bin is not the in $PATH, then it won't work. Or if the user extracted it elsewhere, it won't work too. Maybe we should add a sentence like 

```
If you used the the binary distribution, please use : 
....
Or if you extracted cosi not in `$PATH`, go to the extracted folder
```

something along those lines ?

File: README.md [hunk 2]
```diff
@@ -209,10 +232,35 @@ you will see a "Timeout on signing" or similar error message.
 
 # Standalone Language-specific Verification/Signing Modules
```
Comment (liamsi) [inline README.md:233]: Consistent capitalization (other titles have lowercase letters)

File: README.md (hunk 1 above)
Comment (liamsi) [inline README.md:95]: Not sure. Everything else as described below also won't work in this case (calling `cosi` won't work if it's not in the path)

Comment (ineiti) [inline README.md:95]: I think it should be fine, as we describe in the `binary`-section that it has to go into the PATH somewhere. We can only go so far to help the user...

**Pull request merged**